pcap-dpdk.c

```diff
@@ -0,0 +1,742 @@
+/*
+ * Copyright (C) 2018 All rights reserved.
```

Comment:
Is 2018 the person who holds the copyright to this code or is it an organization?

Or has the person or organization holding the copyright been omitted from the comment?

Reply:
No. This pull request is written by me individually, and I am willing to contribute to libpcap. 
The original comment is copy from other files.
So how should I modify this comment?
Thanks

Reply:
If the code is copyright by you, put your name into the copyright notice, e.g. "Copyright 2018 Jingle Yang All rights reserved."

Do this in other files, such as `pcap-dpdk.h`, as well.

Reply:
OK, Thanks